docs/install-machine.md

```diff
@@ -57,6 +57,27 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+The Machine repository supplies several `bash` scripts that add features such
```

Comment:
Let's move the link to the `contrib/completion/bash` section of the repository to this line, i.e. "supplies several `bash` scripts" should be the link to there.